easybuild/easyconfigs/p/PGI/PGI-16.4-GCC-5.3.0-2.26.eb

```diff
@@ -0,0 +1,25 @@
+name = 'PGI'
+version = '16.4'
+
+homepage = 'http://www.pgroup.com/'
+description = "C, C++ and Fortran compilers from The Portland Group - PGI"
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+sources = ['pgilinux-20%(version_major)s-%(version_major)s%(version_minor)s-x86_64.tar.gz']
+checksums = ['3804dc13b8cf3d50aa16fb56e2682dd7']
+
+gccver = '5.3.0'
+binutilsver = '2.26'
+versionsuffix = '-GCC-%s-%s' % (gccver, binutilsver)
+
+dependencies = [
+    ('GCCcore', gccver),
+    ('binutils', binutilsver, '', ('GCCcore', gccver)),
+    ('numactl', '2.0.11', '', ('GCCcore', gccver)),
+]
+
```

Comment:
The `install_java` bit refers to installing the JRE, nothing else. Using external Javas is "supported" according to https://www.pgroup.com/doc/pgitools.pdf (see "Selecting a Version of Java"). Being said that, I've just tried and it didn't work for me. Java is in the path, and setting PGI_JAVA has no effect. Silly me thinking something this simple would work. I pull back my previous comment, given how problematic something this simple might be, having an installation of Java inside PGI's folder might not be bad.

Related to that: According to https://www.pgroup.com/doc/pgirn.pdf ACML is no longer bundled with PGI, so `install_amd` is meaningless for newer versions (something to handle in the easyblock?). I am not sure at which point they removed it, the document in the link does not reference a particular release (just 2016).